core/src/main/resources/schema/20150602173852_CLOUD-828_stack_-_cluster_relation.sql

```diff
@@ -0,0 +1,18 @@
+-- // CLOUD-828 stack - cluster relation
+-- Migration SQL that makes the change goes here.
+
+ALTER TABLE cluster ADD COLUMN stack_id BIGINT REFERENCES stack(id);
+
+UPDATE cluster SET stack_id=stack.id FROM stack WHERE stack.cluster_id=cluster.id;
+
```

Comment:
We can add a not null constraint here for stack_id, since no cluster shall exist without stack